deploy/awstools/afitools.py

```diff
@@ -13,6 +13,7 @@ def get_fpga_regions():
     fpga_regions = [
         'us-east-1',       # US East (N. Virginia)
         'us-west-2',       # US West (Oregon)
+        'eu-central-1',    # Europe (Frankfurt)
```

Comment:
May you add `eu-west-2` as well to this list (so it matches with the docs).

Reply:
Can you check if `eu-west-2` has F1 instances? If so, we can update this + update the "Setting up..." docs.

Reply:
Right, I somehow missed it. I checked in [EC2 documentation](https://docs.aws.amazon.com/ec2/latest/instancetypes/ec2-instance-regions.html), and it should be supported.